pkg/gofr/datasource/dgraph/dgraph.go

```diff
@@ -17,6 +17,17 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const (
+	bucket1 = 0.05
+	bucket2 = 0.1
+	bucket3 = 0.2
+	bucket4 = 0.5
+	bucket5 = 1.0
+	bucket6 = 2.0
+	bucket7 = 5.0
+	bucket8 = 10.0
+)
+
```

Comment:
Doesn't look right to have buckets named as constant like this

It might be giving magic number linter - but having names like bucket1, bucket2 also doesn't explain anything and would need to go to the code to understand it.

Reply:
Sure, used a slice of float64 instead of these constants in Connect:

```go
	var responseTimeBuckets = []float64{0.05, 0.1, 0.2, 0.5, 1.0, 2.0, 5.0, 10.0}

	// Register metrics
	d.metrics.NewHistogram("dgraph_query_duration", "Response time of Dgraph queries in milliseconds.", responseTimeBuckets...)
	d.metrics.NewHistogram("dgraph_query_with_vars_duration", "Response time of Dgraph "+
		"queries with variables in milliseconds.", responseTimeBuckets...)
	d.metrics.NewHistogram("dgraph_mutate_duration", "Response time of Dgraph mutations in milliseconds.", responseTimeBuckets...)
	d.metrics.NewHistogram("dgraph_alter_duration", "Response time of Dgraph alter operations in milliseconds.", responseTimeBuckets...)

```